ehsm-service/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ehsm-service",
+  "version": "1.0.0",
+  "description": "ehsm-napi-service",
```

Comment:
change the name ehsm-service to ehsm-kms_service

Reply:
It will be change to "ehsm_kms_service".